config/reports.json

```diff
@@ -8,51 +8,59 @@
 		"chrome-ux-report"
 	],
 	"_featured": [
-		"state-of-javascript"
+		"state-of-the-web"
 	],
 	"_metrics": {
 		"bytesCss": {
 			"name": "CSS Bytes",
-			"type": "KB"
+			"type": "KB",
+			"description": "The sum of transfer size kilobytes of all external stylesheets requested by the page. An external stylesheets is identified as a resource with the `css` file extension or a MIME type containing `css`."
 		},
 		"bytesFont": {
 			"name": "Font Bytes",
-			"type": "KB"
+			"type": "KB",
+			"description": "The sum of transfer size kilobytes of all fonts requested by the page. A font is identified as a resource with the `eot`, `ttf`, `woff`, `woff2`, or `otf` file extensions or a MIME type containing `font`."
 		},
 		"bytesHtml": {
 			"name": "HTML Bytes",
-			"type": "KB"
+			"type": "KB",
+			"description": "The sum of transfer size kilobytes of all HTML documents requested by the page. An HTML document is identified as a resource with the `html` or `html` file extensions or a MIME type containing `html`."
 		},
 		"bytesImg": {
 			"name": "Image Bytes",
-			"type": "KB"
+			"type": "KB",
+			"description": "The sum of transfer size kilobytes of all external images requested by the page. An external image is identified as a resource with the `png`, `gif`, `jpg`, `jpeg`, `webp`, `ico`, or `svg` file extensions or a MIME type containing `image`."
 		},
 		"bytesJs": {
 			"name": "JavaScript Bytes",
-			"type": "KB"
+			"type": "KB",
+			"description": "The sum of transfer size kilobytes of all external scripts requested by the page. An external script is identified as a resource with the `js` or `json` file extensions or a MIME type containing `script` or `json`."
 		},
 		"bytesOther": {
 			"name": "Other Bytes",
-			"type": "KB"
+			"type": "KB",
+			"description": "The sum of transfer size kilobytes of all unidentified resources requested by the page. An unidentified resource is one that does not match a known script, font, image, stylesheet, XML file, video, HTML document, or plaintext file."
 		},
 		"bytesTotal": {
 			"name": "Total Kilobytes",
 			"type": "KB",
-			"description": "The sum of <a href=\"https://www.w3.org/TR/resource-timing-2/#dom-performanceresourcetiming-transfersize\">transfer size</a> kilobytes of all resources requested by the page.",
+			"description": "The sum of [transfer size](https://www.w3.org/TR/resource-timing-2/#dom-performanceresourcetiming-transfersize) kilobytes of all resources requested by the page.",
```

Comment:
We only link it here, I assume this is intentional and bytesTotal will be shown as one of the first graphs?

Reply:
Yeah the descriptions were already feeling redundant and I didn't want to repeat the same link 8 times 😛 

I rearranged the metrics so that the totals are first.